case/src/main/kotlin/com/ritense/case/web/rest/CaseDefinitionResource.kt

```diff
@@ -272,6 +274,30 @@ class CaseDefinitionResource(
         }
     }
 
+    @GetMapping("/v1/case/{caseDefinitionName}/hidden-list-column")
+    fun getHiddenCaseListColumnForUser(
+        @LoggableResource("documentDefinitionName") @PathVariable caseDefinitionName: String
+    ): ResponseEntity<List<CaseListColumnDto>> {
+        val currentUserId = UserManagementServiceHolder.currentInstance.currentUserId
```

Comment:
Shouldn't we use the user email?

Reply:
This is the implementation for currentUserId:

```
    @Override
    public String getCurrentUserId() {
        if (SecurityUtils.getCurrentUserAuthentication() != null) {
            return findUserRepresentationByEmail(SecurityUtils.getCurrentUserLogin()).orElseThrow(() ->
                new IllegalStateException("No user found for email: " + SecurityUtils.getCurrentUserLogin())
            ).getId();
        } else {
            return SYSTEM_ACCOUNT;
        }
    }
```
    
So I think this works.